templates/storyboards/swift3.stencil

```diff
@@ -24,89 +20,74 @@ protocol StoryboardSceneType {
 }
 
 extension StoryboardSceneType {
-  static func storyboard() -> UIStoryboard {
-    return UIStoryboard(name: self.storyboardName, bundle: Bundle(for: BundleToken.self))
-  }
-
-  static func initialViewController() -> UIViewController {
-    guard let vc = storyboard().instantiateInitialViewController() else {
-      fatalError("Failed to instantiate initialViewController for \(self.storyboardName)")
-    }
-    return vc
+  static func storyboard() -> {{prefix}}Storyboard {
+    return {{prefix}}Storyboard(name: self.storyboardName, bundle: Bundle(for: BundleToken.self))
   }
 }
 
 extension StoryboardSceneType where Self: RawRepresentable, Self.RawValue == String {
```

Comment:
Looks nice, I like the use of small structs instead of cases.

We're sure that this wouldn't that prevent cases where we actually need to switch over scenes, like maybe switch over `Scene(rawValue: segue.destination.sceneIdentifier)` check which scene is the destination without instantiating a new `UIViewController` when building that `Scene` instance?

tbh as I use the Coordinator pattern in all my projects now, and don't use segues at all anymore, I might be forgetting the patterns and use cases we can encounter when dealing with segues, source & destination VCs and peformSegue implementations…